docs/get-started/get-an-allocation.md

```diff
@@ -73,15 +73,19 @@ On clicking the "Add a project" button will show interface like below:
 
 ![Add A Project](images/coldfront-add-a-project.png)
 
-PIs need to specify appropriate title, description of their research work that you
-will accomplish at NERC in one to two paragraphs and Field of science or research
-domain and then click the "Save" button. Once saved successfully, PIs effectively
-become the "manager" of the project, and are free to add or remove users and also
-request resource allocation(s) to any Projects for which they are the PI. PIs are
-permitted to add users to their group, request new allocations, renew expiring
-allocations, and provide information such as publications and grant data. PIs can
-maintain all their research information under one project or if they require they
-can separate the work into multiple projects.
+!!! danger "Project Title Length Limitation"
+    Please ensure that the project title is both concise and does not exceed a
+    length of 63 characters.
+
+PIs need to specify appropriate title (**lesser than 63 characters**), description
```

Comment:
s/lesser/less/

Reply:
PIs need to specify *an* appropriate title (less than 63 characters), description of their research work that will be performed on the NERC (in one or two paragraphs), the field(s) of science or research domain(s), and then click the "Save" button.